src/ledger/LedgerTxn.cpp

```diff
@@ -1287,37 +1287,111 @@ LedgerTxnRoot::commitChild(EntryIterator iter, LedgerTxnConsistency cons)
     mImpl->commitChild(std::move(iter), cons);
 }
 
+static void
+accum(EntryIterator const& iter, std::vector<EntryIterator>& upsertBuffer,
+      std::vector<EntryIterator>& deleteBuffer)
+{
+    if (iter.entryExists())
+        upsertBuffer.emplace_back(iter);
```

Comment:
maybe a question for later: should we accumulate in something like an `ordered_map` where the key follows the same rough order than the primary key in SQL? I think you mentioned that having things ordered was helping SQL quite a bit as it's touching the b-tree in a less random way

Reply:
Possible (or just sort before inserting) but on closer inspection & discussion with Jon I don't believe my earlier explanation of the performance of the synthetic benchmarks (having to do with order).